src/main/kotlin/no/nav/syfo/handlestatus/HandleStatusInvalid.kt

```diff
@@ -71,10 +71,10 @@ fun handleDuplicateSM2013Content(
             keyValue("originalEdiLoggId", redisSha256String), fields(loggingMeta))
 
     val apprec = fellesformatToAppprec(
-            fellesformat, "Sykmeldingen kan ikke rettes, det må skrives en ny." +
-            "Pasienten har ikke fått beskjed, men venter på ny sykmelding fra deg. Grunnet følgende:" +
+            fellesformat, "Sykmeldingen kan ikke rettes" +
```

Comment:
Dette ser litt rart ut når man skriver det ut sånn som det ser ut for legen i kvitteringen: 
"Sykmeldingen kan ikke rettesPasienten har ikke fått beskjed. Sykmeldingen ble avvist grunnet følgende:Denne sykmeldingen er identisk med ein sykmelding som er mottatt tidligere, og er derfor ein duplikat.Og skal ikke sendes på nytt. Dersom dette ikke stemmer, kontakt din EPJ-leverandør"

Hva om vi heller skriver: 
"Sykmeldingen er avvist fordi den er identisk med en allerede mottatt sykmelding (duplikat), og den kan derfor ikke sendes på nytt. Pasienten har ikke fått beskjed. Kontakt din EPJ-leverandør hvis dette ikke stemmer."

Eller er det sånn at vi må følge et bestemt format der vi starter med hvorvidt sykmeldingen kan rettes, osv..? 

Når er det legene typisk sender samme sykmelding på nytt? Er det fordi de forsøker å rette på noe i en allerede innsendt sykmelding? I så fall kan vi kanskje nevne noe om det i feilmeldingen..?

Reply:
Trenger vi å ta med at pasienten ikke har fått beskjed? Siden det er en duplikat er det implisitt at  brukeren allerede har mottatt denne sykmeldingen. Liker ellers teksten Tia foreslår.

Reply:
Vi trenger det ikkje, men er veldig greit å sei, da dei ofte tenker at nå alltid får beskjed dersom det er ein sykmelding som er avvist

Reply:
Har sett at legene som først får avvist, bare sender inn ein heilt lik ein på nytt.